### PR TITLE
Add macaque 0.7.2

### PR DESCRIPTION
**Pull request by jpdeplaix**

### CODE REVIEW

File: packages/macaque/macaque.0.7.2/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
+opam-version: "1.2"
+name: "macaque"
+version: "0.7.2"
```
Comment (samoht) [inline packages/macaque/macaque.0.7.2/opam]: can you remove the name and version field? thanks!

Comment (jpdeplaix) [inline packages/macaque/macaque.0.7.2/opam]: done

File: packages/macaque/macaque.0.7.2/descr [hunk 2]
```diff
@@ -0,0 +1 @@
+Macaque (Macros for Caml Queries) is a DSL for OCaml, wich produce SQL requests from a comprehension syntax. Macaque can build queries by from simpler components, using phantom types used to ensure safety.
```
Comment (Chris00) [inline packages/macaque/macaque.0.7.2/descr]: You may want that the first line is a short description (≤ 60 chars), the following lines giving possibly more details (wrapped to 80 chars please).

Comment (Chris00) [inline packages/macaque/macaque.0.7.2/descr]: You may want to add the longer description to the field `Description:` of your `_oasis`.

File: packages/macaque/macaque.0.7.2/opam [hunk 3]
```diff
@@ -0,0 +1,31 @@
+opam-version: "1.2"
+license: "LGPL v2 with OCaml linking exception"
+maintainer: "Jacques-Pascal Deplaix <jp.deplaix@gmail.com>"
```
Comment (Chris00) [inline packages/macaque/macaque.0.7.2/opam]: You may want to add this to the `Maintainers:` field of your `_oasis`.

**Comment by samoht** [comment]
Thanks! The long description can be fixed later.

**Pull request merged**